rasgotransforms/rasgotransforms/tests/valid_jinja_test.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from jinja2 import meta
+from jinja2.nodes import FromImport
+import pytest
+
+from rasgotransforms.main import TransformTemplate
+from rasgotransforms.testing_utils import get_all_transform_templates, get_all_macros
+from rasgotransforms.render import RasgoEnvironment
+
+LOGGER = logging.getLogger(__name__)
+MACROS = get_all_macros()
+transforms = get_all_transform_templates()
+
+
+class TestJinja:
+    @pytest.mark.parametrize("transform", transforms, ids=[t.name for t in transforms])
+    def test_transforms_jinja(self, transform: TransformTemplate):
+        environment = RasgoEnvironment(dw_type='snowflake', run_query=None)
```

Comment:
Should we do a parallel run with `environment = RasgoEnvironment(dw_type='bigquery', run_query=None)` ?

Reply:
Good catch, I fixed this to pass in dw_type